aiida/restapi/translator/data/structure.py

```diff
@@ -171,7 +171,14 @@ def get_visualization_data(node, format=None, supercell_factors=[1, 1, 1]):
                             "m": [{"a": atoms_json}],
                             "units": '&Aring;'
                             }
-            response["str_viz_info"]["format"] = "default (ChemDoodle)"
+            response["str_viz_info"]["format"] = "chemdoodle"
+
+        else:
+            try:
+                response["str_viz_info"]["data"] = node._exportstring("xsf")[0]
+                response["str_viz_info"]["format"] = "xsf"
+            except LicensingException as e:
```

Comment:
I'm not sure why are checking for LicensingException here, but this was already there, so I'm discussing this with Snehal as well. For now it's ok to keep it.